cdm/core/src/main/java/ucar/nc2/Group.java

```diff
@@ -376,18 +377,20 @@ public boolean removeAttributeIgnoreCase(String attName) {
 
   /** Find a Enumeration in this or a parent Group, using its short name. */
   @Nullable
-  public EnumTypedef findEnumeration(String name) {
+  public EnumTypedef findEnumeration(String name, boolean searchup) {
```

Comment:
Since this is public, we can't change the method signature. We can add a second method signature though.